app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -4,7 +4,25 @@ name: kickstart_rhsm
 model: ProvisioningTemplate
 snippet: true
 description: |
-  Generate Anaconda RHSM configuration for Red Hat OS registration
+  Generate Anaconda RHSM configuration for Red Hat OS registration available on Red Hat 9 or newer
+
+  Parameters for use with subscription-manager (Red Hat CDN, Satellite, or Katello):
```

Comment:
I wouldn't mention Satellite, Foreman or Katello, just:
`Parameters for use with subscription-manager`

Reply:
Updated to be more generic to anaconda install.